Session/Notifications/PushRegistrationManager.swift

```diff
@@ -340,12 +334,16 @@ public enum PushRegistrationError: Error {
             return
         }
         
+        JobRunner.appDidBecomeActive()
```

Comment:
Is this to ensure messages are sending correctly? Does `JobRunner.appDidFinishLaunching()` end up getting called if the app starts via this entry point? (Just wondering if there are edge-cases we might need to handle by triggering this directly, or if executers that are setup in `AppDelegate` might not have been yet)

Since we are calling `startPollersIfNeeded` below is there anything else in `AppDelegate.handleActivation()` that we should be doing in here that we aren't?

Reply:
Regarding this one, I found that the message sending and receiving jobs were not running at all when app is started by VOIP. `JobRunner.appDidFinishLaunching()` is called but `AppDelegate.handleActivation()` is not called. And I think there are 2 reasons that the message sending and receiving jobs were not running:
1. Job queues were not activated.
2. Job.canRun was set to false because app state is `.inactive`.
Besides, the processes of app launching and handling VOIP are concurrent, we need to do necessary things in the VOIP handling process to make sure and calls can get connected in time.